projects/ng-components/lib/tooltip/tooltip.component.ts

```diff
@@ -15,7 +16,7 @@ import { TooltipTheme, TooltipTrigger, TooltipPosition } from "./tooltip-content
     ],
     encapsulation: ViewEncapsulation.None,
 })
-export class TooltipComponent implements AfterViewInit {
+export class TooltipComponent {
```

Comment:
Why aren't you implementing `AfterViewInit`  considering you are using it ?

Reply:
fixed